Sources/Widgets/Representations/ResliceCursorContextRepresentation/index.js

```diff
@@ -72,6 +72,12 @@ function vtkResliceCursorContextRepresentation(publicAPI, model) {
   model.pipelines.axes.push(axis1);
   model.pipelines.axes.push(axis2);
 
+  model.rotationActorsList = [];
```

Comment:
I am not sure it is useful to store the list in `model`. List can be recomputed from model.pipelines wherever needed. Performance does not seem an issue here.

Reply:
Not sure to understand what you want.

Reply:
wherever model.rotationActorsList is used, instead compute the list from model.pipelines.

was `model.rotationActorsList.includes...` 
now 
```
publicAPI.getRotationActors() {
  return [model.pipelines.axes[0].rotation.actor, ...];
}
...
publicAPI.getRotationActors().includes...
```